packages/event-cache/migrations/20190703025936-add-event-prefix.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('event', 'prefix', {
```

Comment:
Is an index needed ?

Reply:
Yeah, this should probably be indexed.  Will add.

Reply:
sg